setup.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+
+import setuptools
+
+ROOT = os.path.dirname(__file__)
+
+if sys.version_info < (3, 6, 0):
+    sys.exit("Python 3.6.0 is the minimum required version")
+
+with open(os.path.join(ROOT, "README.md")) as f:
+    long_description = f.read()
+
+with open(os.path.join(ROOT, "requirements.txt")) as f:
+    requirements = f.readlines()
```

Comment:
dependencies in ``install_requires`` mustn't be strict, so I think it's not a good choice to use ``requirements.txt`` for that

Reply:
We are building it dynamically. I'm not sure we can strict `install_requires`